cloudant-http/src/main/java/com/cloudant/http/HttpConnection.java

```diff
@@ -109,6 +111,8 @@
     public HttpUrlConnectionFactory connectionFactory = new DefaultHttpUrlConnectionFactory();
 
     private int numberOfRetries = 10;
+    private int attemptNumber = 1;
```

Comment:
@ricellis yes I think you should go with that proposal of `1234abcd-10, then 1234abcd-9 etc`